### PR TITLE
fix : JestResult getSourceAsString Error #280 

### PR DESCRIPTION
**Pull request by developersLxz**
fix : JestResult getSourceAsString Error #280 
the issue is, original code interpret the wrong json struct, it interpret JsonArray as JsonObject.
Thanks :)

### CODE REVIEW

File: jest-common/src/main/java/io/searchbox/client/JestResult.java [hunk 1]
```diff
@@ -122,12 +125,21 @@ public String getSourceAsString() {
...
-        JsonElement obj = jsonObject.get(keys[0]);
-        for (int i = 1; i < keys.length; i++) {
-            obj = ((JsonObject) obj).get(keys[i]);
-        }
-        return obj.toString();
+        
+        StringBuilder sourceStr = new StringBuilder();
+        String pre = "";
+        
+	    JsonElement obj = jsonObject.get(keys[0]);
```
Comment (kramer) [inline jest-common/src/main/java/io/searchbox/client/JestResult.java]: Please adhere to the tab/space usage of the file and project.

Comment (developersLxz) [inline jest-common/src/main/java/io/searchbox/client/JestResult.java]: Sorry , I process the issue now.

**Comment by kramer** [comment]
You got rid of the key iterator logic and that will simply not work for all the actions we have. Try calling `getSourceAsString` on a result of Count request with your changes for example.

I now also see the root problem for #280 which is the iterator logic itself not working for all actions including Search which can return multiple sources...

**Comment by kramer** [comment]
Closing as fixed by cdd62a8.

**Pull request closed without merging**